pypeit/spectrographs/spectrograph.py

```diff
@@ -128,6 +129,14 @@ class Spectrograph:
     team.
     """
 
+    ql_supported = False
```

Comment:
Add this flag to the docs?  See `doc/scripts/build_spectbl_rst.py`.

Reply:
done